src/main/java/com/ghilly/utils/ValidationUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.ghilly.utils;
+
+import java.util.regex.Pattern;
+
+public class ValidationUtils {
+
+    private ValidationUtils() {
+    }
+
```

Comment:
write coment here as format:
// Correct: Bosnia and Herzegovina
// Correct: ,,,
// Incorrect: Russia777
etc.

Reply:
Done